nomenclature/testing.py

```diff
@@ -30,22 +30,22 @@ def assert_valid_yaml(path: Path):
 
 def assert_valid_structure(
     path: Path,
-    dimensions: List[str] = ["region", "variable"],
-    mappings: str = None,
     definitions: str = "definitions",
+    mappings: Optional[str] = None,
+    dimensions: Optional[List[str]] = None,
 ) -> None:
     """Assert that `path` can be initialized as a :class:`DataStructureDefinition`
 
     Parameters
     ----------
     path : Path
-        directory path to the file of interest
-    dimensions : List[str], optional
-        List of dimensions to be checked
-    mappings : str, optional
-        Name for the mappings folder, defaults to "mappings"
+        Project directory to be validated
     definitions : str, optional
-        Name for the definitions folder, defaults to "definitions"
+        Name of the definitions folder, defaults to "definitions"
+    mappings : str, optional
+        Name of the mappings folder, defaults `mappings` (if this folder exist)
```

Comment:
Sorry, my bad, I was confused in my formatting in the suggestion earlier - we should use "" for a proper name and `` for the argument...
```suggestion
        Name of the mappings folder, defaults to "mappings" (if this folder exist)
```